ui/src/app/components/cases/steps-list.component.ts

```diff
@@ -283,7 +283,7 @@ export class StepsListComponent extends BaseComponent implements OnInit {
   bulkDestroy() {
     this.testStepService.bulkDestroy(this.selectedStepsList).subscribe({
       next: () => {
-        this.fetchTestCase(this.route.parent.snapshot.params.testCaseId);
+        this.fetchTestCase(this.route.parent.snapshot.params.testCaseId || this.testCase.id);
```

Comment:
@manikanta-reddy-t Is there a need for both this.route.parent.snapshot.params.testCaseId and this.testCase.id ? Do we know in which cases these one of these 2 are valid ?

Reply:
this.testcase.id is sufficient, removing the first case

Reply:
done